scripts/unsupported/revert_sam/SamToUnmappedBams.wdl

```diff
@@ -0,0 +1,150 @@
+task CramToBam {
+  File? input_cram
+  String output_bam_name
+  File? ref_fasta
+  File? ref_fasta_index
+  Int? disk_size
+
+  command {
+    samtools view -1 -h -T ${ref_fasta} -o ${output_bam_name} ${input_cram}
+  }
+
+  runtime {
+    docker: "broadinstitute/genomes-in-the-cloud:2.2.3-1469027018"
```

Comment:
does a samtools only docker image exist?

Reply:
It would be great if it did, do you know where I could find one? I can make one but then it won't look official.